ci-operator/config/openshift/ceph-csi/openshift-ceph-csi-master.yaml

```diff
@@ -4,6 +4,9 @@ build_root:
 images:
 - dockerfile_path: ocs/Containerfile
   to: ceph-csi
+promotion:
+  name: ocs-stream
+  namespace: ocp
```

Comment:
This makes the tooling think you are part of OCP and thus test if integrating your image with OCP into a release breaks (the `--target=[release:latest]` addition in the images presubmits).

Either add a tag_specification specifying which OCP versioon to test against, or promote to a different namespace.

Reply:
@petr-muller can we just use a namespace without creating it in advance? It would make sense to use `ocs` in this case.

Reply:
`ocs` namespace even exists today (but you can use a new namespace too, it will get created)